probe_scraper/probe_expiry_alert.py

```diff
@@ -158,16 +159,28 @@ def create_bug(probes: List[ProbeDetails], version: str, api_key: str) -> int:
     probe_names = [probe.name for probe in probes]
     probe_prefix = get_longest_prefix(probe_names, tolerance=1)
 
+    see_also_bugs = list(set([probe.previous_bug for probe in probes
+                              if type(probe.previous_bug) == int]))
```

Comment:
nit: `isinstance`

Reply:
Oops I completely blanked when I was trying to remember that